requirements/bootstrap.txt

```diff
@@ -0,0 +1,22 @@
+#
+# Securely installing `pip-compile` for regenerating dependencies.
+#
+# This file is generated with `pip-compile` using the command:
+#
+#    echo "pip-tools" | pip-compile --allow-unsafe --generate-hashes \
+#      --output-file=requirements/bootstrap.txt -
```

Comment:
Edited them manually, because I don't expect them to change. I've seen CUSTOM_COMPILE_COMMAND. Do you propose to add a `make requirements/bootstrap.txt` to it?

Reply:
Added `make` target to regenerate the file.